controllers/dbController.js

```diff
@@ -0,0 +1,79 @@
+import express from 'express'
+import cors from 'cors'
+import morgan from 'morgan'
+import Student from '../models/Student.js';
+import '../connect-db.js'
+
+const app = express()
+
+app.use( express.json())
+app.use( cors() ) 
+app.use( morgan("dev") ) 
```

Comment:
The declaration of middleware (json, cors, morgan) you actually did already in server.js and actually it is suficient to declare it once :) 
And because you just create & export the controllers here, you actually do not even need to import express here at all.